astropy/coordinates/tests/test_frames.py

```diff
@@ -187,27 +186,19 @@ def test_frame_repr():
     i3 = ICRS(ra=1*u.deg, dec=2*u.deg, distance=3*u.kpc)
 
     assert repr(i2) == ('<ICRS Coordinate: (ra, dec) in deg\n'
-                        '    ({})>').format(' 1.,  2.' if NUMPY_LT_1_14
-                                             else '1., 2.')
+                        '    ({})>').format('1., 2.')
```

Comment:
Here and below, the change is of course is correct, but there isn't really a reason any more to even format the data: just insert `1., 2.` in the string proper...

Reply:
good point